content/events/contributor-summit/index.adoc

```diff
@@ -13,12 +13,15 @@ links:
 The Jenkins Contributor Summit brings together current and future contributors to the Jenkins project.
 The summit brings together community members to learn, meet, and help shape the future of Jenkins.
 
-=== Contributor Summit Feb 23-25, 2021
+=== Online Contributor Summit on June 25, 2021
 
-See the Feb 23-25, 2021 link:https://www.jenkins.io/blog/2021/02/16/contributor-summit-online/[summit announcement] for more details.
+The next contributor summit will take place on June 25, 2020.
```

Comment:
timetraveling ❤️ 
```suggestion
The next contributor summit will take place on June 25, 2021.
```

Reply:
Yeah. Don't forget to buy bitcoins and Tesla stocks